src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java

```diff
@@ -664,20 +665,13 @@ protected RepositoryData callCreateRepoApi(String projectKey, Repository repo)
                   + " - no response from endpoint, please check logs",
               repo.getName(), projectKey));
     }
-    setRepositoryAdminPermissions(data, projectKey, ID_GROUPS, repo.getUserGroup());
-    setRepositoryWritePermissions(data, projectKey, ID_USERS, technicalUser);
-    if (grantRepositoryWriteToAllOpenDevStackUsers) {
-      logger.info(
-          "Grant write to every member of {} to repository {}",
-          openDevStackUsersGroupName,
-          data.getSlug());
-      setRepositoryPermissions(
-          data.getSlug(),
-          projectKey,
-          ID_GROUPS,
-          openDevStackUsersGroupName,
-          REPOSITORY_PERMISSIONS.REPO_WRITE);
+    if (!project.isSpecialPermissionSet()) {
+      String adminGroup = repo.getAdminGroup();
+      if (adminGroup != null && !adminGroup.isEmpty()) {
```

Comment:
It looks to me that this logic can be moved to the method callCreateProjectApi because the permission should be inherit from the bitbucket project instead to be set explicity in at repository level. What do you think?

Reply:
If we do that, the default admin group will also get admin rights on the project itself. This should be agreed by Irene.

Reply:
> the default admin group will also get admin rights on the project itself. 

I noticed that is happening anyway:  See line method `createComponentRepositoriesForODSProject`:
https://github.com/opendevstack/ods-provisioning-app/blob/544ce527c6101fa238eff5e249b634b9d6765277/src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java#L476-L482

Reply:
As we have agreed, this idea is discarded due to backward compatibility issues.